chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/urluptime/DatadogSyntheticsTestService.java

```diff
@@ -110,7 +110,7 @@ public SyntheticsAPITest createSyntheticsTest(String url, List<Long> developerId
                                         .type(SyntheticsAssertionType.STATUS_CODE)),
                                 new SyntheticsAssertion(new SyntheticsAssertionTarget()
                                         .operator(SyntheticsAssertionOperator.MATCHES)
-                                        .target("/[\\S\s]+[\\S]+/")
+                                        .target("/[\\S]/")
```

Comment:
I'm thrown by the regex not having the `+` after the `[\S]`, though I don't think I'm against it. Just want to make sure it's what we want